src/pages/HomePage.jsx

```diff
@@ -1,11 +1,16 @@
-import React from 'react'
+import React from "react";
+import NoteState from "../context/NoteState";
+import InputForm from "../comonents/InputForm";
 
 function HomePage() {
   return (
-    <div>
-      
+    <div style={{width:"60%",margin:'auto'}}>
```

Comment:
remove inline css, create seperate file for css and attach

Reply:
Resolved sir